src/scss/style.scss

```diff
@@ -19,4 +19,4 @@
 @import 'components/_testButton';
 @import 'components/_logo';
 @import 'components/_mainImage';
-@import 'components/_buttonhalloffame';
+@import 'components/_hallOfFameButton';
```

Comment:
W tym pliku coś ktoś pomieszał chyba przy marge. 
Np. Twoja 9 i 22 linijka to to samo. Tak samo 7 i 19.
Jakbyś mógł to przy okazji uporządkuj ten plik tak:
```
@import 'externalImports';
@import 'reset';
@import 'vars';
@import 'main';

// * LAYOUTS
@import 'src/scss/layouts/mainWrapper';
@import 'src/scss/layouts/logoSection';
@import 'src/scss/layouts/chooseGameSection';
@import 'src/scss/layouts/mainSection';

// * COMPONENTS
@import 'src/scss/components/testButton';
@import 'src/scss/components/newGameButton';
@import 'src/scss/components/hallOfFameButton';
@import 'src/scss/components/logo';
@import 'src/scss/components/mainImage';
```

Reply:
tak, jak z pulla to wszystko się podwoiło, nie widziałem czy to zostało celowo zrobione czy przy pullu coś się rozjechało. już poprawiam